lib/movie/screens/movie_screen.dart

```diff
@@ -1,7 +1,25 @@
 import 'package:flutter/material.dart';
 import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:movie/infrastructure/theme/app_colors.dart';
+import 'package:movie/movie/domain/movie.dart';
+import 'package:movie/movie/domain/movie_service.dart';
 import 'package:movie/movie/screens/cubit/movie_cubit.dart';
-import 'package:movie/movie/screens/cubit/movie_state.dart';
+import 'package:movie/movie/screens/drawer_menu_widget.dart';
+
+// ---Texts---
+const _kTitle = 'Movie';
+
+// ---Parameters---
+const _kPadding = EdgeInsets.all(10);
```

Comment:
pls, extract only values from this widget.

Expl: `const double _kPadding = 10.0`